package.json

```diff
@@ -8,18 +8,35 @@
     "IPFS",
     "files"
   ],
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.es5.js",
-  "browser": "dist/index.umd.min.js",
+  "main": "lib/index.js",
+  "exports": {
+    ".": "./lib/index.js",
+    "./*": "./lib/*.js",
+    "./package.json": "./package.json"
```

Comment:
Is this referring to the `package.json` in the `src`? If so why doesn't this have `src` or `lib` in the path?

Reply:
No this is actually referring to the root `package.json` file. The reason for this line is such that you can do this:
```js
import pkg from "webnative/package.json`
```

If we wouldn't have this line, it would try to load `webnative/lib/package.json.js`.

It's not *really* necessary to expose the package.json, it's just kinda useful, sometimes.